### ISSUE TITLE
Interactive Init

### ISSUE THREAD

**Issue by Joebeazelman** [alire-project/alire#1204]
Alire supports installing the toolchain interactively from its command-line, but not for initializing the repository. It's odd since new users are more likely to need help initializing the repository since it is one of the most common operations. This can be done using the same command-line option of the toolchain.

**Comment by Fabien-Chouteau** [comment]
Indeed, the nim package manager has a really good interactive init we should get some inspiration from there.

**Comment by mosteo** [comment]
Bringing here the suggestion in #1203:

> issuing `alr edit` after the crate has been initialized, assuming the default editor is also configured.

Would be a last question after interactive initialization.
